app/assets/stylesheets/geoblacklight/modules/item.scss

```
@@ -45,13 +45,40 @@
     -webkit-line-clamp: 8;
   }
 
-  #table-container{
-    overflow:scroll;
-    max-height: 450px;
+  #table-container {
     margin-bottom: 1rem;
 
     table {
       margin-bottom: 0px;
     }
   }
+
+  .sidebar-container {
+    width: 25vw;
+    overflow: scroll;
```

```suggestion
    overflow: scroll;
    overscroll-behavior: contain;
```
I noticed that when scrolling in the sidebar, you could cause the underlying page to scroll if you scrolled past the end, which felt kind of strange. This seemed to fix it